pkg/api/customization/clusterregistrationtokens/import.go

```diff
@@ -30,7 +30,7 @@ func ClusterImportHandler(resp http.ResponseWriter, req *http.Request) {
 		authImage = authImages[0]
 	}
 
-	if err := systemtemplate.SystemTemplate(resp, image.Resolve(settings.AgentImage.Get()), authImage, token, url); err != nil {
+	if err := systemtemplate.SystemTemplate(resp, image.Resolve(settings.AgentImage.Get()), authImage, token, url, false); err != nil {
```

Comment:
Why always set it false here?

Reply:
For now, we could not support to import a Windows cluster, as we could not deploy the wins on the Windows host.